Student/StudentUnitTests/.swiftlint.yml

```diff
@@ -9,6 +9,7 @@ disabled_rules: # rule identifiers to exclude from running
   - xctfail_message
   - type_body_length
   - file_length
+  - weak_delegate
```

Comment:
this seems fine in tests, but i wonder if there could arise a situation where being warned about this could help figure out a test.  Would it be better to add a `// swiftlint:disable:next weak_delegate` for the specific case you're trying to treat?

Reply:
Fine with me. Fixed.